tests/specs/utils/http.spec.js

```diff
@@ -114,18 +114,22 @@ describe('HTTP', () => {
             });
         });
 
+        function randomIntBetween(min, max) {
+            return Math.floor(Math.random() * (max - min)) + min;
+        }
+
         describe('#handleStatus', () => {
-            for (let status = 200; status < 300; status++) {
+            [200, randomIntBetween(201, 299), 299].forEach((status) => {
```

Comment:
The test output was bothering me. Iterating over every status code is unnecessary and actually introduced a few seconds slowdown.

Before:
```
 602 passing (3s)
```

After:
```
122 passing (423ms)
```

But boundary values + random compromise s a bit random. maybe I should just get rid of the random tests?